pull_request_template.md

```diff
@@ -1,9 +1,28 @@
-Test plan - (Please fill in how you tested your changes)
+## Description
+<!---Describe your changes in detail-->
 
-Please make sure your submission complies with our [Development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [Formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), and [Commit Message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests) guidelines. Don't forget to follow our [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution) for any code copied from other projects.
+## Motivation and Context
+<!---Why is this change required? What problem does it solve?-->
+<!---If it fixes an open issue, please link to the issue here.-->
 
-Fill in the release notes towards the bottom of the PR description.
-See [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) for details.
+## Impact
+<!---Describe any public API or user-facing feature change or any performance impact-->
+
+## Test Plan
+<!---Please fill in how you tested your change-->
+
+## Release Notes
+Please follow [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes towards the bottom of the PR description.
+
+## Contributor's checklist
+
+- [ ] Please make sure your submission complies with our [Development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [Formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), and [Commit Message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests) guidelines.
+- [ ] Followed [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution) for any code copied from other projects.
+- [ ] PR description addresses the issue accurately and concisely
+- [ ] Documented new properties (with its default value), SQL syntax, functions, or other functionality 
+- [ ] Follows [Release Notes Guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
```

Comment:
please use word consistently, either `Follows` or `Followed`, and others.

Reply:
+1